src/Utilities/Utilities.jl

```diff
@@ -363,15 +363,34 @@ end
 
 # Finding URLs -- based partially on code from the main Julia repo in `base/methodshow.jl`.
 
+function in_cygwin()
+    # check if in cygwin, if so we need to translate cygwin paths to windows paths
+    if is_windows()
+        try
+            success(`cygpath -h`)
+        catch
+            return false
+        end
+    else
+        return false
+    end
+end
+
 function url(repo, file)
     file = abspath(file)
     remote = getremote(dirname(file))
     isempty(repo) && (repo = "https://github.com/$remote/tree/{commit}{path}")
     commit, root = cd(dirname(file)) do
-        readchomp(`git rev-parse HEAD`), readchomp(`git rev-parse --show-toplevel`)
+        toplevel = readchomp(`git rev-parse --show-toplevel`)
+        if in_cygwin()
```

Comment:
Agreed. However, I think I did come up with another special case: when the packages are installed under Cygwin (by changing `JULIA_PKGDIR`), then the paths in the docsystem are
```
julia> Docs.docstr(Docs.Binding(Documenter,:Documenter)).data[:path] |> print
\home\morten\julia\v0.6\Documenter\src\Documenter.jl
```
but `git rev-parse --show-toplevel` + `cygpath` adds a drive

```
shell> git rev-parse --show-toplevel
/cygdrive/c/home/morten/julia/v0.6/Documenter

shell> cygpath -m /cygdrive/c/home/morten/julia/v0.6/Documenter
C:/home/morten/julia/v0.6/Documenter
```
which should make `startswith` fail.